lib/logstash/inputs/udp.rb

```diff
@@ -39,35 +39,47 @@ class LogStash::Inputs::Udp < LogStash::Inputs::Base
 
   HOST_FIELD = "host".freeze
 
-  public
   def initialize(params)
     super
     BasicSocket.do_not_reverse_lookup = true
-  end # def initialize
+    @output_queue = nil
+  end
 
-  public
   def register
     @udp = nil
-  end # def register
+  end
 
-  public
   def run(output_queue)
-  @output_queue = output_queue
+    @output_queue ||= output_queue
```

Comment:
if `@output_queue` is already set, we will ignore the explicitly-given value?

I'm not sure I see the value of this change (and its mate on L45 where we initialize `@output_queue` with a `nil` value).

Reply:
yeah, I had a brain bubble for this one. For some reason I tough that the `run` method could be called multiple time and this change juste make sure we set `@output_queue` just once. in either ways both ways works but this change is useless. will revert, thanks for catching this.